### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by jlarmstrongiv** [vadimdemedes/pastel#64] [General]
I just wanted to say thank you for creating [`pastel`](https://www.npmjs.com/package/pastel)

I’ve used [`commander`](https://www.npmjs.com/package/commander), [`yargs`](https://www.npmjs.com/package/yargs), [`oclif`](https://www.npmjs.com/package/oclif), and others, but nothing comes close to the ease of use of [`pastel`](https://www.npmjs.com/package/pastel) for creating scripts for my projects

I went ahead and added a [PR](https://github.com/colinhacks/zod/pull/2949) to add it to the [`zod`](https://www.npmjs.com/package/zod) ecosystem so that more people can find it

Thanks again 🎉

**Comment by vadimdemedes** [comment]
Thank you, John, appreciate the kind words 💛
